DataGrid/Gantt: Rewrite PDF troubleshooting topic (#7630)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/15 PDF Export Issues/00 Export Unicode Characters/00 DataGrid/00 DataGrid.md

When exporting DataGrid data to PDF, some characters may be displayed incorrectly in the generated file. 

> PDF supports a limited set of fonts and they are all limited to the ASCII-codepage. If you want to use UTF-8 characters, you need to supply a font that includes necessary glyphs. This tutorial demonstrates how to add custom fonts to your DataGrid PDF exports.

Reviewer: Just checking, but is "ASCII-codepage" a widely-used technical term?

Author: ASCII is certainly widely-known, but I'm not 100% sure about "codepage". Perhaps we can replace it with something a touch simpler like "characters"?
```suggestion
PDF supports a limited set of fonts and they are all limited to the ASCII characters. If you want to use UTF-8 characters, you need to supply a font that includes necessary glyphs. This tutorial demonstrates how to add custom fonts to your DataGrid PDF exports.
```

Reviewer: I agree, "ASCII characters" sounds better to me. I'd like to make a few changes:

```suggestion
PDF supports a set of fonts that are limited to the ASCII characters. If you want to use UTF-8 characters, you need to supply a font that includes the required glyphs. This tutorial demonstrates how to add custom fonts to your DataGrid PDF exports.
```
